config/mysql.go

```diff
@@ -0,0 +1,34 @@
+package config
+
+import (
+	"log"
+	"os"
+
+	"github.com/joho/godotenv"
+	"gorm.io/driver/mysql"
+	"gorm.io/gorm"
+)
+
+var DB *gorm.DB
```

Comment:
```suggestion
var db *gorm.DB
```
đặt tên viết thường để ở ngoài không gọi trực tiếp vào db đc, ảnh hưởng config ko debug đc nếu có lỗi
viết thêm 1 hàm GetDb để trả ra rồi xài